projects/ngneat/cashew/src/lib/httpCacheLocalStorage.ts

```diff
@@ -0,0 +1,53 @@
+import { HttpResponse } from '@angular/common/http';
+import { Inject, Injectable } from '@angular/core';
+import { deleteByRegex } from './deleteByRegex';
+import { HttpCacheStorage } from './httpCacheStorage';
+import { HTTP_CACHE_CONFIG, HttpCacheConfig } from './httpCacheConfig';
+
+@Injectable()
```

Comment:
There is no need to read from `localstorage` each time. Create an in-memory cache, and use it after you fetched the value once. For example:

```ts
cache = new DefaultHttpCacheStorage();

get(key: string) {
  return this.cache.get(key) || this.getLocalStorage()[key];
}
```